kernel/exec.c

```diff
@@ -31,11 +31,23 @@
 #include <myst/spinlock.h>
 #include <myst/strings.h>
 #include <myst/syscall.h>
+#include <myst/syslog.h>
 #include <myst/tcall.h>
 #include <myst/thread.h>
 
 #define GUARD 0x4f
 
+#define GLIBC_THREAD_STACK_SIZE_MIN 1048576 /* 1mb */
+
+#define THREAD_STACK_SIZE_WARNING \
+    "\n\
+    The thread stack size may be too small for the given program interpreter,\n\
```

Comment:
Is "program interpreter" equivalent to "linker loader"?

Reply:
The interpreter is /lib64/ld-linux-x86-64.so.2 for glibc.

Reply:
ELF has PT_INTERP program segment that contains the name of what ELF calls the "program interpreter" (which they call that in readelf dumps).